xml/System.ServiceModel.Syndication/SyndicationTextInput.xml

```diff
@@ -126,9 +174,16 @@
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets or sets the title of the textInput feed item.</summary>
```

Comment:
`textInput` is not a parameter or a type, it seems to be metadata (please correct me if I'm wrong). If that's the case, then it would make sense to wrap it with `<c><c/>` (ignore the suggestion if it's not metadata):

```suggestion
        <summary>Gets or sets the title of the <c>textInput</c> feed item.</summary>
```

Reply:
Same suggestion would apply to all the similar keywords if the metadata condition applies for them.